node/commons/models/VtexSeller.ts

```diff
@@ -34,19 +34,21 @@ class VtexSeller {
       AvailableQuantity: 10000,
       discountHighlights: [],
       teasers: [],
-      Installments: installment && [
-        {
-          Value: installment.value,
-          InterestRate: 0,
-          TotalValuePlusInterestRate: price,
-          NumberOfInstallments: installment.count,
-          Name: '',
-        },
-      ],
+      Installments: [],
```

Comment:
```suggestion
      Installments: installment ? [
        {
          Value: installment.value,
          InterestRate: 0,
          TotalValuePlusInterestRate: price,
          NumberOfInstallments: installment.count,
          Name: '',
        },
      ] : [],
```

just a suggestion, but with that you won't need the if below :)

Reply:
Done! Thanks 👌